src/osd/OSD.cc

```diff
@@ -5536,6 +5541,28 @@ void OSD::_send_boot()
   set_state(STATE_BOOTING);
 }
 
+std::string OSD::_collect_compression_algorithms()
+{
+  const auto& compression_algorithm_names = Compressor::compression_algorithm_names;
+  const auto& plugin_registry = cct->get_plugin_registry()->plugins;
+
+  if(0 == plugin_registry.size())
```

Comment:
please use `empty()` instead, if what you are interested is the emptiness instead of of the actual size.

Reply:
+1